clearing-house/src/lib.rs

```diff
@@ -620,7 +620,7 @@ pub mod pallet {
         /// alludes to the amount of collateral that can be withdrawn without making the account go
         /// below the initial margin ratio.
         ///
-        /// ![](https://www.plantuml.com/plantuml/svg/FOux3i8m40LxJW47IBQdYeJ4FJREmxPahwtzmFM9AAX6CzLivgmUlLrkLLAB0w7jMjodtOcKFskkNc8FWwOX3l4rZKwFqUSmtXkUbT9V29OAb5xA7PGQMAlafOmmq54vdzr8qSSRIsVDLTRPM7u76-Gu-GK0)
+        /// ![](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/escher-exchange/escher/main/clearing-house/diagrams/withdraw-collateral.plantuml)
```

Comment:
I was wondering, is it worth to upgrade all links to `https` ?

Reply:
I don't think there's any sensitive information we need to protect in this case (we're just downloading an image)

Reply:
Fair enough :+1: